### PR TITLE
Feature/en50 add unit tests for add word dictionary processor

### PR DESCRIPTION
**Pull request by A-Romany**

### CODE REVIEW

File: src/test/java/org/agbrothers/englishbot/processing/processor/dictionary/AddWordDictionaryProcessorTest.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+        ArgumentCaptor<Word> captor = ArgumentCaptor.forClass(Word.class);
+        verify(dictionaryService).addWord(captor.capture());
+        assertEquals("sea", captor.getValue().getEnglishValue());
+
+        addWordDictionaryProcessor.process(exchange2);
+        String result2 = exchange2.getResponseMessageText();
+        assertEquals("Слово fly - муха було додано до словника.", result2);
+
+        ArgumentCaptor<Word> captor1 = ArgumentCaptor.forClass(Word.class);
```
Comment (AgienkoSergiy) [inline src/test/java/org/agbrothers/englishbot/processing/processor/dictionary/AddWordDictionaryProcessorTest.java]: you can use same `captor` that is on line 51

File: src/test/java/org/agbrothers/englishbot/processing/processor/dictionary/AddWordDictionaryProcessorTest.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        addWordDictionaryProcessor.process(exchange3);
+
+        String result1 = exchange1.getResponseMessageText();
+        assertEquals("Помилка в слові англійською або перекладі слова. Слово англійською має бути латиницею, переклад - лише кирилицею.", result1);
+
+        String result2 = exchange2.getResponseMessageText();
+        assertEquals("Помилка в слові англійською або перекладі слова. Слово англійською має бути латиницею, переклад - лише кирилицею.", result2);
+
+        String result3 = exchange3.getResponseMessageText();
+        assertEquals("Помилка в слові англійською або перекладі слова. Слово англійською має бути латиницею, переклад - лише кирилицею.", result3);
```
Comment (AgienkoSergiy) [inline src/test/java/org/agbrothers/englishbot/processing/processor/dictionary/AddWordDictionaryProcessorTest.java]: Extract `"Помилка в слові...` to String variable

**Review by AgienkoSergiy — CHANGES_REQUESTED** [review]

File: src/test/java/org/agbrothers/englishbot/processing/processor/dictionary/AddWordDictionaryProcessorTest.java (hunk 2 above)
Comment (A-Romany) [reply src/test/java/org/agbrothers/englishbot/processing/processor/dictionary/AddWordDictionaryProcessorTest.java]: Thanks. It's much better

File: src/test/java/org/agbrothers/englishbot/processing/processor/dictionary/AddWordDictionaryProcessorTest.java (hunk 1 above)
Comment (A-Romany) [reply src/test/java/org/agbrothers/englishbot/processing/processor/dictionary/AddWordDictionaryProcessorTest.java]: Thanks. You're right.

**Review by AgienkoSergiy — DISMISSED** [review]

**Review by AgienkoSergiy — APPROVED** [review]

**Pull request merged**